### ISSUE TITLE
🆕 Software Suggestion | InviZible Pro

### ISSUE THREAD

**Issue by StrongAntelope** [privacytoolsIO/privacytools.io#1586]
## Basic Information

**Name:** InviZible Pro
**Category:** Android Privacy Addons
**URL:** https://github.com/Gedsh/InviZible

## Description

**InviZible Pro** is a all-in-one tool that supports DNSCrypt, Tor and Purple I2P, all of which (known as modules) can be run at the same time. It also allows selection of websites and apps that can bypass Tor. There are other advace features like selection of dns servers, various ports, tor bridge, etc. The only requirement is root permission. If root isn't available, DNSCrypt and Tor can be used as proxies with NetGuard or individually with the applications that supports proxy.

**Comment by blacklight447-ptio** [comment]
Unfortunately, we cannot recommend this app because of its root requirement, its the same reason why we don't recommend AFwall+ and adaway. Rooting is disastrous for your androids security.

**Comment by StrongAntelope** [comment]
> Unfortunately, we cannot recommend this app because of its root requirement

Dev is currently looking into a VPN solution. Hopefully this will arrive in a few month.

**Comment by StrongAntelope** [comment]
@blacklight447-ptio: Support for non-root users added in the latest binaries. Take a look here: https://github.com/Gedsh/InviZible/wiki/Proxy-mode-(no-root-required)

**Comment by Mikaela** [comment]
How does InviZible Pro configure Tor ports and stream isolation? What part of the code does that?

**Comment by lrq3000** [comment]
I strongly recommend to add it, it works surprisingly well, and indeed without the need for root now. It allows to connect to both Tor and I2P networks simultaneously, from any browser. I don't know if other apps than browsers are also tunneled but I would assume so since by default it's the "VPN mode" that is enabled, although it also offers a "Proxy mode" so that we can tunnel per app by setting a proxy (I guess SOCKS5?) per app that supports proxies.

**Comment by HACKERALERT** [comment]
I strongly agree with @lrq3000 as well that InviZible Pro should be added to PrivacyTools. It doesn't require root and can work in both proxy and VPN mode, and allows a combination of Tor, I2P, and DNSCrypt in one package. And it's highly configurable. This is something that is extremely hard to find for Android, which is why I believe it would be a valuable addition.
